docker-paradox.toml

```diff
@@ -0,0 +1,11 @@
+[general]
+port = 3030
+public = false
```

Comment:
This should be set to `true`, which is bind to `0.0.0.0` vs `false` which is bind to `127.0.0.1`